superset-frontend/src/dashboard/components/nativeFilters/FilterBar.tsx

```diff
@@ -0,0 +1,337 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  QueryFormData,
+  styled,
+  SuperChart,
+  t,
+  ExtraFormData,
+} from '@superset-ui/core';
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import cx from 'classnames';
+import { Form } from 'src/common/components';
+import Button from 'src/components/Button';
+import Icon from 'src/components/Icon';
+import FilterConfigurationLink from './FilterConfigurationLink';
+// import FilterScopeModal from 'src/dashboard/components/filterscope/FilterScopeModal';
+
+import {
+  useCascadingFilters,
+  useFilterConfiguration,
+  useSetExtraFormData,
+} from './state';
+import { Filter } from './types';
+import { getChartDataRequest } from '../../../chart/chartAction';
+import { areObjectsEqual } from '../../../reduxUtils';
+
+const barWidth = `250px`;
+
+const BarWrapper = styled.div`
+  width: ${({ theme }) => theme.gridUnit * 6}px;
+  &.open {
+    width: ${barWidth}; // arbitrary...
+  }
+`;
+
+const Bar = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  /* bottom: 100%;
+  right: 100%; */
+  flex-direction: column;
+  flex-grow: 1;
+  width: ${barWidth}; // arbitrary...
+  background: ${({ theme }) => theme.colors.grayscale.light5};
+  border-right: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+  height: 100%;
+  max-height: 100%;
+  display: none;
+  /* &.animated {
```

Comment:
On the fence about deleting all this commented animation stuff vs keeping it for potential future use.

Reply:
it's not hurting anybody 🤷